packages/e2e-test-utils-playwright/src/editor/site-editor.ts

```diff
@@ -23,6 +23,37 @@ export async function saveSiteEditorEntities( this: Editor ) {
 	// not have that classname.
 	// TODO - find a way to improve this selector to use role/name.
 	await this.page.waitForSelector(
-		'.edit-site-save-button__button:not(.is-busy)'
+		'css=.edit-site-save-button__button:not(.is-busy)'
+	);
+}
+
+/**
+ * Toggles the global styles sidebar (opens it if closed and closes it if open).
+ *
+ * @param {Editor} this
+ */
+export async function toggleGlobalStyles( this: Editor ) {
+	await this.page.click( 'role=button[name="Styles"i]' );
+}
+
+/**
+ * Opens a global styles panel.
+ *
+ * @param {Editor} this
+ * @param {string} panelName Name of the panel that is going to be opened.
+ */
+export async function openGlobalStylesPanel( this: Editor, panelName: string ) {
+	const selector = `xpath=//div[@aria-label="Settings"]//button[.//*[text()="${ panelName }"]]`;
```

Comment:
We usually want to use role selectors over other selectors. Furthermore, I think we can inline this as well if we convert it to use role selectors.